xdsl/dialects/riscv.py

```diff
@@ -317,6 +337,11 @@ def print_parameter(self, printer: Printer) -> None:
         printer.print_string_literal(self.data)
 
 
+# TODO workaround as reviewers do not like external dependency for typing.Self
```

Comment:
Please remove comment, and open a separate PR for the typing.Self. I think we should add it, but it deserves a focused discussion

Reply:
This TODO is important, to avoid left over code just like in current xdsl code base where ```TypeVar``` declaration is everywhere but some of them no longer be used anywhere.